skbase/tests/test_base.py

```diff
@@ -177,6 +177,7 @@ def __init__(self, foo, bar=84):
         self.foo = foo
         self.foo_ = deepcopy(foo)
         self.bar = bar
+        super().__init__()
```

Comment:
I think we should test for that this is caught.

In `sktime` it is caught by checking whether `_tags_dynamic` is initialized.

Reply:
I think that makes sense. Will verify whether we have the test and if not add it as part of #62.